src/execsnoop/main.rs

```diff
@@ -0,0 +1,231 @@
+extern crate bcc;
```

Comment:
No `extern crate` needed for any of these dependencies in Rust 2018. Please remove.